bin/autoUpdateRepo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#:Title: autoUpdateRepo.sh
+#:Author: Alan Coleman
+
+#changes directory to folder containing repositories
+cd ~/gitfolder
+
+#runs repository update script 
+bash ../ELK_POC_HOME/elk-poc/GitUpdateScript.sh
```

Comment:
should this be pointing to an ENV variable e.g. $ELK_POC_HOME/elk-poc/...

Reply:
Environmental variables currently don't work when the script is ran from crontab. This will be addressed soon as a new task on asana.